### DISCUSSION TITLE
Change placeholder text

### DISCUSSION THREAD

**Discussion by nicholasvperry** [thecodejack/svelte-file-dropzone#160] [Ideas]
Is there anyway to change the place holder text? I only want to accept one file so I want the placeholder to say something else. Like "Drag 'n' drop a file here, or click to select file"

**Comment by bgrundeken** [comment]
The drop zone has a slot, so you can put a child component inside with the custom text.

See [this example](https://github.com/thecodejack/svelte-file-dropzone/blob/master/stories/views/CustomSlotDropzoneView.svelte).

**Comment by arackaf** [comment]
I’ll assume this solves it - if not feel free to re-open with more detail :-)

**Comment by nicholasvperry** [comment]
This is exactly what I wanted. Thank you!
